augly/video/functional.py

```diff
@@ -443,11 +443,9 @@ def color_jitter(
     func_kwargs = helpers.get_func_kwargs(metadata, locals(), video_path)
 
     color_jitter_aug = af.VideoAugmenterByColorJitter(
-        brightness_level=brightness_factor,
-        contrast_level=contrast_factor,
-        saturation_level=saturation_factor,
+        brightness_factor, contrast_factor, saturation_factor
```

Comment:
```suggestion
        brightness_level=brightness_factor,
        contrast_level=contrast_factor,
        saturation_level=saturation_factor,
```
Using keywords to name args doesn't hurt, might help avoid errors later if the order were to be switched :)